src/app/components/observation/observation-image-carousel/observation-image-carousel.component.html

```diff
@@ -0,0 +1,82 @@
+<ng-container>
+  <swiper-container
+    #swiper
+    slides-per-view="1"
+    loop="false"
+    css-mode="false"
+    navigation="true"
+    pagination="false"
+    space-between="0"
+    centered-slides="false"
+    center-insufficient-slides="false"
+    zoom="true"
+    keyboard="false"
+    (swiperslidechange)="onSlideChange($event)"
```

Comment:
Det er lettere å forså hva som skjer i templaten om metoden som kalles skifter navn til det som faktisk skjer, feks:

```suggestion
    (swiperslidechange)="setCurrentSlideIndex($event.detail[0].activeIndex)"
```

Så lenge metoden bare gjør én ting, så er det mer lesbart, trenger ikke trykke seg inn i js-koden for å se hva som skjer.

Eventuelt bare

```suggestion
    (swiperslidechange)="index.set($event.detail[0].activeIndex)"
```

Dersom clickedAttachmentIndex skifter navn til index..

Reply:
det går ikke å bruke $event.detail[0] i templaten. jeg får i hvert fall feil.
![image](https://github.com/user-attachments/assets/ee68bbee-8ce3-43c9-a1da-791d19e6ee61)